src/components/Header/WhatNew.tsx

```
@@ -162,13 +150,11 @@
             li={[
               <>
-                add post{' '}
-                <DialogClose asChild>
-                  <Link href="/articles/dropped-eslint-plugin-sort-keys-custom-order-from-eslint-config-ts-prefixerv1120">
-                    Dropped eslint-plugin-sort-keys-custom-order from
-                    eslint-config-ts-prefixer@v1.12.0
-                  </Link>
-                </DialogClose>
+                add post{` `}
+                <Link href="articles/dropped-eslint-plugin-sort-keys-custom-order-from-eslint-config-ts-prefixerv1120">
+                  Dropped eslint-plugin-sort-keys-custom-order from
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)